grammar/GlslLexer.flex

```diff
@@ -99,6 +99,8 @@ INCLUDE_PATH={IDENTIFIER}([\s\/]*{IDENTIFIER}\s*)*(\.{IDENTIFIER})?
 
 <MACRO_IDENTIFIER_STATE> {
   {WHITE_SPACE}                    { return WHITE_SPACE; }
+  {BACKSLASH}                      { return WHITE_SPACE; }
```

Comment:
Can you please explain more about this one? And also can you add steps to reproduce? I'm saying that because I cannot even remember how many nights I spent to make macros work in this plugin and this touches the core of the logic so I'm very very afraid to touch anything that is related to macros

Reply:
It's for handling multiline macros
e.g.
```glsl
#define SETPOS(pos, transform) { \
vec4 _tmp_ = pos * transform; \
gl_Position = _tmp_; \
}
```
it's not a 100% perfect solution, but in almost every case a `\` in shader macros will only every appear for escaping the line terminator (given how strings are not really a thing), so it's more or less a tradeoff. Fixes a semi-common edge case while adding a very rare one.

Reply:
```
#define \
EXAMPLE 10
```